tests/test_config.py

```diff
@@ -0,0 +1,83 @@
+# pylint: disable=W0621, C0116
+
+import pytest
+
+from fideslib.core.config import (
+    DatabaseSettings,
+    SecuritySettings,
+    config,
+    get_config,
+    load_toml,
+)
+from fideslib.exceptions import MissingConfig
+
+
+@pytest.fixture
+def config_dict(fides_toml_path):
+    yield load_toml([fides_toml_path])
+
+
+def test_database_settings_sqlalchemy_database_uri_str(config_dict):
+    expected = "postgresql://someuri:216f4b49bea5da4f84f05288258471852c3e325cd336821097e1e65ff92b528a@db:5432/test"  # pylint: disable=C0301
```

Comment:
Personally, I'd rather just see `line-too-long` disabled globally in `pyproject.toml`. It's arbitrary, and `black` handles the formatting for us anyway.
```suggestion
    expected = "postgresql://someuri:216f4b49bea5da4f84f05288258471852c3e325cd336821097e1e65ff92b528a@db:5432/test"
```

It's also referenced on L29.